### PR TITLE
Facilitate app deployment using a CF manifest file #68

### PR DESCRIPTION
**Pull request by atwoosnam**

### CODE REVIEW

File: src/ViewModels/src/DeploymentDialog/DeploymentDialogViewModel.cs [hunk 1]
```diff
@@ -36,6 +36,8 @@ public class DeploymentDialogViewModel : AbstractViewModel, IDeploymentDialogVie
         private CloudFoundryInstance _selectedCf;
         private CloudFoundryOrganization _selectedOrg;
         private CloudFoundrySpace _selectedSpace;
+        private string manifestPathLabel;
```
Comment (ccheetham) [inline src/ViewModels/src/DeploymentDialog/DeploymentDialogViewModel.cs:39]: not sure what the naming convention is, however these names are inconsisten. Some have _ prefix and some not.

**Review by ccheetham — CHANGES_REQUESTED** [review]
fix var names

File: src/ViewModels/src/DeploymentDialog/DeploymentDialogViewModel.cs (hunk 1 above)
Comment (atwoosnam) [reply src/ViewModels/src/DeploymentDialog/DeploymentDialogViewModel.cs]: Ah, yeah I think this is a product of using the "Convert to full property" feature in VS; it creates a private var to track the state of the prop you're converting but doesn't abide by our naming conventions apparently 😅 

Resolved in 66d1f23

**Review by ccheetham — APPROVED** [review]
lgtm

**Pull request merged**